bouncer/src/repo/manipulator/modelconvertor/import/odaHelper/data_processor_rvt.cpp

```diff
@@ -936,15 +1213,15 @@ std::unordered_map<std::string, std::string> DataProcessorRvt::fillMetadata(OdBm
 {
 	std::unordered_map<std::string, std::string> metadata;
 	metadata[REVIT_ELEMENT_ID] = std::to_string((OdUInt64)element->objectId().getHandle());
+
 	try
 	{
 		fillMetadataByElemPtr(element, metadata);
 	}
 	catch (OdError& er)
 	{
-		repoInfo << "Caught exception whilst trying to fetch metadata by element pointer " << convertToStdString(er.description());
-
-		repoTrace << "Caught exception whilst trying to fetch metadata by element pointer " << convertToStdString(er.description());
+		repoInfo << "Caught exception whilst trying to fetch metadata by element pointer " << convertToStdString(er.description()) << " Code: " << er.code();
+		repoTrace << "Caught exception whilst trying to fetch metadata by element pointer " << convertToStdString(er.description()) << " Code: " << er.code();
```

Comment:
I think we can remove this line... not sure why we print it twice on 2 different severity 😆

Reply:
Done!